google/cloud/pubsub/internal/extend_leases_with_retry.cc

```diff
@@ -56,6 +57,32 @@ google::pubsub::v1::ModifyAckDeadlineRequest UpdateRequest(
   return request;
 }
 
+class ExtendLeasesWithRetryImpl {
+ public:
+  virtual ~ExtendLeasesWithRetryImpl() = default;
+  virtual future<Status> MakeRpc(
```

Comment:
Perhaps `MakeRpc()` might be better as `AsyncModifyAckDeadline()`?

Reply:
Done